### ISSUE TITLE
Let's have it: 0.9.4 vs. master

### ISSUE THREAD

**Issue by StanAngeloff** [jashkenas/coffeescript#816]
This is the place where we put **0.9.4** in the same room as **master** and have them exchange punches until one of them falls down. 

Please give +1 if you are a big fan or **master** and wish to punch **0.9.4** real hard.

Please give -1 if you think your good pal **0.9.4** is still alive and kicking and is better than **master**.

You may explain the reasoning behind your vote, but please be brief and to the point, i.e. nail it.

So basically I want to see how you feel about the recent changes being introduced and what is the general view of the direction CoffeeScript is headed to. Simple enough, right?

-- Stan out
